terra-jupyter-aou/Dockerfile

```diff
@@ -110,6 +110,15 @@ RUN curl -SL --output dotnet.tar.gz "https://dotnetcli.azureedge.net/dotnet/Runt
 
 RUN echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
```

Comment:
nit: would probably put this below the bcftools install, just to keep package setup somewhat separate from environment configuration